analyses/verbose/sarif_helper.go

```diff
@@ -0,0 +1,153 @@
+package verbose
+
+import (
+	"errors"
+	"fmt"
+	"github.com/fabric8-analytics/cli-tools/analyses/driver"
+	"github.com/owenrumney/go-sarif/models"
+	"github.com/owenrumney/go-sarif/sarif"
+	"github.com/rs/zerolog/log"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strings"
+)
+
+type RegexDependencyLocator struct {
+	FileContent         string
+	Ecosystem           string
+	EndIndices          []int
+	DependencyNodeIndex []int
+}
+
```

Comment:
```suggestion
// ProcessSarif -- a valid comment can be added--
```
ditto for other exported members